### PR TITLE
 Update database to apply RealID selection to the appropriate card, w…

### PR DESCRIPTION
**Pull request by LeDeep**
…hether ID or DL

### CODE REVIEW

File: server/models/client-data-parser.js [hunk 1]
```diff
@@ -253,15 +253,18 @@ function extractCardOptions(data) {
...
     })
-    if(data.realID.getRealID === 'Yes') {
-      cardOptions.push({
-        type:               'DL',
-        option_type:        'modification',
-        option_value:       'real-id'
-      })
-    }
   }
+
```
Comment (baccigalupi) [inline server/models/client-data-parser.js]: How does this get associated with the DL card? Seems like you need to associate it with the right card_id at this point. But maybe I am not understanding the code. Can you explain?

File: server/db/migrations/20171211140452_updateCardIDTypeToInteger.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.alterTable('card_options', function(table) {
+    table.integer('card_id').alter();
+  });
+};
+
```
Comment (baccigalupi) [inline server/db/migrations/20171211140452_updateCardIDTypeToInteger.js]: Is this fixing a mistake in the type of the id as it was originally setup?

Comment (LeDeep) [reply server/db/migrations/20171211140452_updateCardIDTypeToInteger.js]: Yes. The card_id was originally a string, but it needs to be an integer.

File: server/models/client-data-parser.js (hunk 1 above)
Comment (LeDeep) [reply server/models/client-data-parser.js]: This happens in the server_data_parser.js, where it assigns the real ID designation to the selected card type, when card.id === card_id.

**Pull request merged**